README.md

```diff
@@ -18,6 +21,20 @@ Livros recomendados pelos participantes das comunidades DevAraraquara / OpenSanc
 * **Comentários:**
     * Um livro que deve estar presente com todos desenvolvedores que prezam por qualidade de software. Um livro com ótimos conselhos e exemplos de como e por que devemos escrever um código limpo e como evitar os "maus cheiros" de nosso código.
 
+## Programação Nível Básico
+
+### Ruby
```

Comment:
@rberaldo change to use this way and later we can check how use different anchors on it.